code/view/json.php

```diff
@@ -342,8 +326,10 @@ protected function _getEntityRoute(ModelEntityInterface $entity)
      */
     protected function _convertRelativeLinks(ViewContextInterface $context)
     {
-        if (is_array($this->_content) || $this->_content instanceof \Traversable) {
-            $this->_processLinks($this->_content);
+        $content = $context->content;
```

Comment:
@johanjanssens you might wanna copy this as reference if $context->content can ever be a pure array.